src/plugins/intel_cpu/src/nodes/scaled_attn.cpp

```diff
@@ -505,6 +509,139 @@ struct MHAKernel<ScaledDotProductAttention::KT_ONEDNN, T> {
+
+            float* q_ptr = &query.at<float>({b, h, m_start, 0});
+            float* k_ptr = &present_key.at<float>({b, h / h_each_group_len, 0, 0});
+            float* v_ptr = &present_value.at<float>({b, h / h_each_group_len, 0, 0});
+
+            float* alibi_ptr = nullptr;
+            auto alibi_stride = 0;
+            if (alibi_mask) {
+                alibi_ptr = &alibi_mask.at<float>({b, h, 0, 0}, true);
+                if (alibi_mask.size(2) > 1)
+                    alibi_stride = alibi_mask.stride(2);
+            }
+            uint8_t* attn_mask_ptr = nullptr;
+            auto attn_mask_stride = 0;
+            if (attention_mask) {
+                attn_mask_ptr = reinterpret_cast<uint8_t*>(&attention_mask.at<float>({b, h, 0, 0}, true));
+                if (attention_mask.size(2) > 1)
+                    attn_mask_stride = attention_mask.stride(2) * sizeof(float);
+            }
+            uint8_t* cmask_ptr = nullptr;
+            auto cmask_stride = 0;
+            if (causal_mask) {
+                cmask_ptr = &causal_mask.at<uint8_t>({b, h, 0, 0}, true);
+                if (causal_mask.size(2) > 1)
+                    cmask_stride = causal_mask.stride(2);
+            }
+
+            arm_compute::TensorInfo qkInfo;
+            arm_compute::Tensor qkTensor;
+
+            bool b_transpose = false;
+            if (k_stride_s == 1)
+                b_transpose = true;
+            GemmKernel qk_gemm_ptr(m_cnt, head_size, kv_len, b_transpose);
+
+            qk_gemm_ptr.executeGemm(reinterpret_cast<void *>(q_ptr),
+                                    reinterpret_cast<void *>(k_ptr),
+                                    qkInfo,
+                                    qkTensor);
+
+            auto qk = reinterpret_cast<float*>(qkTensor.buffer());
+
+
+            for (size_t m = m_start; m < m_end; m++) {
+                // apply attention mask & sofmax
+                auto ncausal = auto_causal ? (kv_len - q_len + m + 1) : kv_len;
+                attn_softmax(qk + (m - m_start) * kv_len,
+                             qk + (m - m_start) * kv_len,
+                             d_scale,
+                             alibi_ptr + m * alibi_stride,
+                             attn_mask_ptr + m * attn_mask_stride,
+                             cmask_ptr + m * cmask_stride,
+                             select_nfltmax_at_0,
+                             ncausal,
+                             kv_len,
+                             ov::element::f32,
+                             ov::element::f32);
+            }
+            arm_compute::TensorInfo outInfo;
+            arm_compute::Tensor outTensor;
+
+            auto out = has_out_transpose ? &output_emb.at<float>({b, m_start, h * head_size}) : &output_emb.at<float>({b, h, m_start});
+            auto strides = arm_compute::Strides({output_emb.stride_bytes(1), output_emb.stride_bytes(2)});
+            GemmKernel out_gemm_ptr(m_cnt, kv_len, head_size);
```

Comment:
NIT
```suggestion
            GemmKernel out_gemm(m_cnt, kv_len, head_size);
```
This is an object, not a pointer.